nixos/modules/services/backup/restic.nix

```diff
@@ -256,6 +256,24 @@ in
             having to manually specify most options.
           '';
         };
+
+        niceness = mkOption {
+          description = lib.mdDoc "Niceness for instances of restic. Also applies to remote ones connecting via ssh when positive. Only used for `restic backup`, `restic forget` and `restic check` commands.";
+          type = types.ints.between (-20) 19;
+          default = 10;
+        };
+
+        ioSchedulingClass = mkOption {
+          description = lib.mdDoc "IO scheduling class for restic (see ionice(1) for a quick description). Applies to local instances, and remote ones connecting by ssh if set to idle. Only used for `restic backup`, `restic forget` and `restic check` commands.";
+          type = types.enum [ "idle" "best-effort" "realtime" ];
+          default = "best-effort";
```

Comment:
Same as for my comment regarding the `niceness` default. Maybe default to `none`?